third_party/2and3/cryptography/fernet.pyi

```diff
@@ -5,13 +5,21 @@ class InvalidToken(Exception): ...
 class Fernet(object):
     def __init__(self, key: Union[bytes, Text]) -> None: ...
     def decrypt(self, token: bytes, ttl: Optional[int] = ...) -> bytes: ...
+    # decrypt_at_time accepts None ttl at runtime but it's an implementtion detail and it doesn't really
+    # make sense for the client code to use it like that, so the parameter is typed as int as opposed to
+    # Optional[int].
+    def decrypt_at_time(self, token: bytes, ttl: int, current_time: int) -> bytes: ...
```

Comment:
Shouldn't `ttl` be `Optional`? After all, it's passed directly to `decrypt`. Or are there other reasons why using `None` here wouldn't work?